public/js/train.js

```diff
@@ -207,7 +207,7 @@ $(document).ready(function() {
   $('.classifier input[type=file]').on('change', function(e) {
     var nameInput = $(e.target).parent().find('input[type=text]');
     if ($(e.target).length > 0 && ($(e.target)[0].files && $(e.target)[0].files.length > 0)) {
-      var validMimeType = {'application/zip': true}[$(e.target)[0].files[0].type];
+      var validMimeType = {'application/zip': true, 'application/x-zip-compressed': true}[$(e.target)[0].files[0].type];
```

Comment:
This line feels a little confusing -  it might be worth extracting to an `isZip()` function or something like that.